parsers/CENACE.py

```diff
@@ -128,13 +128,10 @@ def fetch_csv_for_date(dt, session: Session | None = None):
     response.raise_for_status()
 
     # API returns normally status 200 but content type text/html when data is missing
-    if (
-        "Content-Type" not in response.headers
-        or response.headers["Content-Type"] == "text/html"
+    if "Content-Type" not in response.headers or "text/html" in response.headers.get(
+        "Content-Type"
     ):
-        raise Exception(
-            f"Error while fetching csv for date {datestr}: No CSV was returned by the API. Probably the data for this date has not yet been published."
-        )
+        return None
```

Comment:
```suggestion
    if (
        "Content-Type" not in response.headers
        or response.headers["Content-Type"] == "text/html"
    ):
        return None
```
Otherwise it trows a type errors as it's trying to compare the literal string with `str | None`. Changing the fallback would probably also work.

Reply:
You meant the `raise` `return` ? Also, If I remember correctly `response.headers["Content-Type"]` is an array and can have more values, that's why I used a `in`

Reply:
Pylance is reporting the following:
```
Operator "in" not supported for types "Literal['text/html']" and "str | None"
Operator "in" not supported for types "Literal['text/html']" and "None"
```
This was not a problem before.

Reply:
Let me take a look at that 🤔

Reply:
Done!